Content.Shared/Alert/AlertCategory.cs

```diff
@@ -12,6 +12,7 @@ public enum AlertCategory
     Health,
```

Comment:
I made a uh, new alert for the Silicon thing. You should check how I formatted that one, I don't remember if it was like this or not.